src/applications/personalization/profile/components/ProfileWrapper.jsx

```diff
@@ -10,23 +10,30 @@ import NameTag from '~/applications/personalization/components/NameTag';
 import ProfileSubNav from './ProfileSubNav';
 import ProfileMobileSubNav from './ProfileMobileSubNav';
 import { PROFILE_PATHS } from '../constants';
-import { EditContainer } from './edit/EditContainer';
-import { routesForNav } from '../routesForNav';
+import { ProfileFullWidthContainer } from './ProfileFullWidthContainer';
+import { getRoutesForNav } from '../routesForNav';
+import { selectProfileToggles } from '../selectors';
+import { normalizePath } from '../../common/helpers';
 
-// default layout includes the subnavs
-// edit layout is a full-width layout
 const LAYOUTS = {
-  DEFAULT: 'default',
-  EDIT: 'edit',
+  SIDEBAR: 'sidebar',
+  FULL_WIDTH: 'full-width',
 };
 
-// we want to use a different layout for the edit page
-// since the profile wrapper is getting passed in the router as children
-// we can really scope a layout to just the edit page in a more 'react router' way
-const getLayout = currentPathname => {
-  return currentPathname === PROFILE_PATHS.EDIT
-    ? LAYOUTS.EDIT
-    : LAYOUTS.DEFAULT;
+// we want to use a different layout for the specific routes
+// the profile hub and edit page are full width, while the others
+// include a sidebar navigation
+const getLayout = ({ currentPathname, profileUseHubPage }) => {
+  const path = normalizePath(currentPathname);
+
+  const pathLayoutMap = {
+    [PROFILE_PATHS.EDIT]: LAYOUTS.FULL_WIDTH,
+    [PROFILE_PATHS.PROFILE_ROOT]: profileUseHubPage
+      ? LAYOUTS.FULL_WIDTH
+      : LAYOUTS.SIDEBAR,
+  };
+
+  return pathLayoutMap[path] || LAYOUTS.SIDEBAR;
```

Comment:
I kind of like this idea of using object literal lookups instead off switch statements: 

https://www.30secondsofcode.org/js/s/switch-object/#:~:text=Luckily%2C%20JavaScript's%20object%20literals%20are,pass%20to%20the%20switch%20statement.

I figured this would also make it easier to add more layouts in the future.